R/build.R

```diff
@@ -164,7 +164,7 @@ desctable.default <- function(data, stats = stats_auto, tests, labels = NULL)
   # Build the complete table
   list(Variables = varColumn(data, labels),
     stats = statTable(data, stats)) %>%
-  `class<-`("desctable")
+  `class()<-`("desctable")
```

Comment:
The name of the function *is* `class<-`, no need for parentheses as they are already there, surrounding `"desctable"`